layouts/partials/matomo.html

```diff
@@ -0,0 +1,21 @@
+{{ $siteId := cond (isset .Site.Params "matomositeid") .Site.Params.MatomoSiteId "1" }}
+{{ $clientSideDNT := cond (isset .Site.Params "matomoclientsidednt") .Site.Params.MatomoClientSideDNT true }}
```

Comment:
Lets do the DNT check just like in the google analytics template https://raw.githubusercontent.com/gohugoio/hugo/master/tpl/tplimpl/embedded/templates/google_analytics.html and introduce a param like `matomoRespectDoNotTrack`. If that option is `true` it should not even include the tracking snippet, just like in the google analytics.  Also be sure to check for user DNT setting in browser `var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);`.